Tests/MongoSwiftSyncTests/MongoCollectionTests.swift

```diff
@@ -272,6 +272,28 @@ final class MongoCollectionTests: MongoSwiftTestCase {
         expect(deleteManyResult).to(beNil())
     }
 
+    func testRenamed() throws {
+        let encoder = BSONEncoder()
+        let client = try MongoClient.makeTestClient()
+        let db = client.db("testRenamedDB")
+        defer { try? db.drop() }
+        let coll = try db.createCollection("testRenamedColl")
```

Comment:
to ensure the rename actually worked, we should insert some stuff into this collection first, then rename it, then do a count on the new collection and ensure the documents moved.